static/js/containers/auth/LoginPasswordPage.js

```diff
@@ -102,9 +99,10 @@ const mapStateToProps = state => {
   const authFlow = getAuthFlowSelector(state)
   const partialToken = getAuthPartialTokenSelector(state)
   const formError = getFormErrorSelector(state)
-  const email = getAuthEmailSelector(state)
-  const name = getAuthNameSelector(state)
-  const profileImageUrl = getAuthProfileImageSelector(state)
+  const authUserDetail = state.ui.authUserDetail || {}
```

Comment:
Is there a reason you moved away from selector usage here? `R.path` will return `undefined`, which is what you'll get for `name`, `email`, and `profileImageUrl` with this code too.

Reply:
No real reason. I'll add it